### PR TITLE
Add adventure files for robot help

### PR DESCRIPTION
**Pull request by Mornayvdm1**
Adding my adventures in addition to the happy-robot story line.

### CODE REVIEW

**Comment by Mornayvdm1** [comment]
Please review this, @casey-collab

File: english/happy-robot/happy-robot.md [hunk 1]
```diff
@@ -2,3 +2,7 @@ A happy robot helps people.
 
 It was a dark and stormy night.  A scream is heard.  A happy robot, let's name him "George," wakes up to the sound.
 "Someone needs my help!"
+
+He jumps up, but before he rushes out the door he...
+a) Grabs a [weapon](robot-help/robot-weapon.md), or
```
Comment (casey-collab) [inline english/happy-robot/happy-robot.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/happy-robot/happy-robot.md [hunk 2]
```diff
@@ -2,3 +2,7 @@ A happy robot helps people.
...
+b) Enjoys some [refreshments](robot-help/refreshments.md).
```
Comment (casey-collab) [inline english/happy-robot/happy-robot.md]: This link is also invalid.

Comment (casey-collab) [inline english/happy-robot/happy-robot.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

File: english/robot-help/refreshments.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+The happy robot thinks he might need some 'assistance' to help anyone that might be in trouble.
```
Comment (casey-collab) [inline english/robot-help/refreshments.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: english/robot-help/robot-weapon.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+Now what shall the happy robot need if there is a possible fight, to defend himself?
```
Comment (casey-collab) [inline english/robot-help/robot-weapon.md]: This file has also been added to a directory that already contained a page.

File: english/robot-help/refreshments.md (hunk 3 above)
Comment (casey-collab) [inline english/robot-help/refreshments.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: english/robot-help/robot-weapon.md (hunk 4 above)
Comment (casey-collab) [inline english/robot-help/robot-weapon.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by Mornayvdm1** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :smile:

**Pull request merged**